src/wallet/rpcdump.cpp

```diff
@@ -676,6 +700,10 @@ UniValue importwallet(const JSONRPCRequest& request)
 
 UniValue dumpprivkey(const JSONRPCRequest& request)
 {
+    if (IsMultiwalletJSONRPCRequest(request)) {
+        return ExecForeachWalletJSONRPCRequest(request, &dumpprivkey);
+    }
```

Comment:
Something like `dumpprivkey` should probably just return the *first* result it gets (and ignore all the failures)?